db/migrate/20150708153926_create_authorizations.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :authorizations do |t|
       t.belongs_to :user, index: true
       t.belongs_to :o_auth_application, index: true
-      t.string :refresh_token
+      t.string :refresh_token, index: {unique: true, length: 191}
```

Comment:
Yeah I forgot we aren't in the middle of that massive PR where I was free to redo the migrations each time :+1: